site/_scss/blocks/_post-navigation.scss

```diff
@@ -0,0 +1,86 @@
+.post-navigation {
+  border-bottom: 1px solid var(--color-hairline);
+  border-top: 1px solid var(--color-hairline);
+  display: flex;
+  font-family: 'Google Sans';
+  justify-content: space-between;
+  margin-bottom: px-to-rem(-72px);
+  margin-top: px-to-rem(80px);
+  width: 100%;
+
+  @include media-query('md-max') {
+    flex-direction: column;
+  }
+
+  &__link {
+    display: block;
+    text-decoration: none;
+    width: 50%;
+
+    @include media-query('md-max') {
+      width: 100%; 
+    }
+  }
+
+  &__next,
+  &__prev {
+    font-size: px-to-rem(20px);
+    height: 100%;
+    line-height: px-to-rem(28px);
+    padding: px-to-rem(40px);
+
+    @include media-query('lg-max') {
+      padding: px-to-rem(40px) px-to-rem(24px); 
+    }
+  }
+
+  &__prev {
+    border-right: 1px solid var(--color-hairline);
+
+    @include media-query('md-max') {
+      border-bottom: 1px solid var(--color-hairline);
+      border-top: unset; 
+    }
+  }
+  
+  &__next:hover,
+  &__prev:hover {
+    background-color: var(--color-blue-lightest);
+  }
+  
+  &__label {
+    color: var(--color-primary);
+    display: inline-block;
+    font-size: px-to-rem(14px);
+    line-height: px-to-rem(20px);
+    margin-bottom: px-to-rem(16px);
+  }
+
+  &__label::before {
+    background-position: center center;
+    background-repeat: no-repeat no-repeat;
+    background-size: cover;
+    content: '';
+    display: inline-block;
+    height: px-to-rem(16px);
+    margin-right: px-to-rem(20px);
+    margin-top: px-to-rem(-2px);
+    vertical-align: middle;
+    width: px-to-rem(16px);
+  }
+
+  &__next &__label::before {
```

Comment:
Nit: make this `&__label:before` and then only flip in `&_prev &__label:before` with `transform` - saves some bytes in the CSS.